plugins/pom.xml

```diff
@@ -33,6 +33,7 @@
                 <module>org.locationtech.udig.libs.tests</module>
                 <module>org.locationtech.udig.location.test</module>
                 <module>org.locationtech.udig.render.feature.basic.test</module>
+				<module>org.locationtech.udig.render.gridcoverage.basic.test</module>
```

Comment:
Looks like tabs are at the beginning of the lin rather that spaces, can we fix it?